app/views/shared/_nav.html.erb

```diff
@@ -1,8 +1,8 @@
 <nav class="navbar navbar-dark navbar-expand-md bg-dark" role="navigation">
-  <div class="container">
-    <%= link_to root_url, class: 'navbar-brand' do %>
-    <%= inline_svg_tag 'pod_logo.svg', height: 24, width: 80, alt: '', class: 'd-inline-block align-text-top' %>
-    <span class="ms-1"><%= t('.brand') %></span>
+  <div class="container" style="align-items: last baseline!important">
```

Comment:
Is there a reason to do this inline vs. in one of the stylesheets? I don't think you need `!important` for this to work.